### PR TITLE
update llmc export

### PR DESCRIPTION
**Pull request by Qazalin**
Fixing the optimizer, device API and ADAM self.t, I use this for scheduler benchmarking.

### CODE REVIEW

File: examples/llm.c/export.py [hunk 1]
```diff
@@ -62,7 +63,7 @@
     if v.lazydata.base.buffer not in used_buffers: print(f"UNUSED: {k}")
     if v.grad is not None: grad_state_dict['grad_'+k] = v.grad
   state_dict.update(grad_state_dict)
-  state_dict.update({'adam_b1': optimizer.b1, 'adam_b2': optimizer.b2, 'adam_t': optimizer.t, 'adam_lr': optimizer.lr})
+  state_dict.update({'adam_b1': optimizer.b1_t, 'adam_b2': optimizer.b2_t, 'adam_lr': optimizer.lr})
```
Comment (chenyuxyz) [inline examples/llm.c/export.py:66]: are you sure you want to update `adam_b1` with `b1_t`? `bt_t` tracks `b1 ** t`

Comment (Qazalin) [reply examples/llm.c/export.py:66]: b1 and b2 are float literals now - They don't have an independent schedule

Comment (chenyuxyz) [reply examples/llm.c/export.py:66]: i see, update the key to `adam_b1_t` and `adam_b2_t` to match?
